src/include/firebird/FirebirdInterface.idl

```diff
@@ -599,6 +599,8 @@ version:	// 3.0 => 4.0
 	// Batch API
 	Batch createBatch(Status status, Transaction transaction, uint stmtLength, const string sqlStmt,
 		uint dialect, MessageMetadata inMetadata, uint parLength, const uchar* par);
+
+	uint getRemoteProtocolVersion(Status status);
```

Comment:
@AlexPeshkoff can't this be a method of another thing, for example IUtil receiving the attachment as parameter? It does not look good for me in IAttachment... Also, looks like a layer violation. Maybe something about consult capabilities information would be much better to consult in gbak.

Reply:
Do not see layer violation - but agree that this call is looking bad here. Avoid it in favor of 'good old' getFbVersion().